pyhuman/app/workflows/open_office_calc.py

```diff
@@ -0,0 +1,90 @@
+import os
+import random
+import pyautogui
+from lorem.text import TextLorem
+from time import sleep
+from ..utility.base_workflow import BaseWorkflow
+
+
+WORKFLOW_NAME = 'OpenOfficeCalc'
+WORKFLOW_DESCRIPTION = 'Create spreadsheets with Apache OpenOffice Calc (Windows)'
+DEFAULT_WAIT_TIME = 2
+OPEN_OFFICE_PATH = "C:\Program Files (x86)\OpenOffice 4\program\soffice"
+
+def load():
+    return OpenOfficeCalc()
+
+class OpenOfficeCalc(BaseWorkflow):
+
+    def __init__(self, default_wait_time=DEFAULT_WAIT_TIME, open_office_path=OPEN_OFFICE_PATH):
+        super().__init__(name=WORKFLOW_NAME, description=WORKFLOW_DESCRIPTION)
+        self.default_wait_time = default_wait_time
+        self.open_office_path = open_office_path
+
+    def action(self, extra=None):
+        self._create_spreadsheet()
+
+    def _create_spreadsheet(self):
+        self._new_spreadsheet()
+        self._move_to_cell([random.choice('abcde'),random.randrange(6)]) # move to random cell, given column & row parameters
+        sleep(1)
+        self._insert_table()
+        sleep(1)
+        self._move_to_cell([random.choice('abcdefghijkl'),random.randrange(15)]) # move to random cell, given column & row parameters
+        self._insert_comment()
+        sleep(3)
+        self._save_quit()
+
+    def _insert_comment(self):
+        pyautogui.hotkey('ctrl', 'alt', 'c') # insert comment
+        pyautogui.typewrite(TextLorem().sentence()) # type random sentence
+        pyautogui.press('esc') # finish commenting
+        sleep(self.default_wait_time)
+
+    def _find(self):
```

Comment:
Is this method necessary to have in this workflow? Doesn't seem like it's used in this class